### PR TITLE
ignorePaths getting ignored

### PR DESCRIPTION
**Pull request by ab77**
Set in https://github.com/balena-io/balena-cloud/pull/1153 instead.

### CODE REVIEW

**Comment by klutchell** [comment]
Can you ELI5?

**Comment by ab77** [comment]
> Can you ELI5?

The %% in our dockerfile templates break renovate (makes it skip repos i.e balena-cloud) and the `ignorepaths` stanza isn'y making it into the resolved repo config.

**Comment by ab77** [comment]
> Yeah, that would probably do it. I did the same thing [here](https://github.com/balena-os/meta-balena/blob/master/.github/renovate.json).

OK, that should do it then: https://github.com/balena-io/balena-cloud/pull/1153
